### DISCUSSION TITLE
I'm trying to understand radius and increment.

### DISCUSSION THREAD

**Discussion by dragon3025** [Zodsmar/SeedSearcherStandaloneTool#94] [Q&A] [answered]
What is the measurement unit of radius, blocks or chunks?

Also does increment skip every other block within this radius or does cause radius to multiply?

**Comment by Zodsmar** [comment, marked as answer]
Radius is from 0,0 how far out in each direction (+, +) and (-, -) in blocks.
The increment is how many blocks to skip so 1 incrementer is a 1 block skip its not multiplied.

**Reply by FoxBlood75** [reply]
I'm curious if it skips around the perimeter like a dotted line, out from the center making a 'donut' or skips around the radius area as a whole creating (for lack of a better description) a solid circle of dotted search points?

I've seen some comments on YT looking for a way to search for islands, so the 'donut' search ignoring what's at the center may be kind of a cool option...

**Reply by brycycle111** [reply]
FoxBlood, if you look at the [nested for loops in the BiomeSearcher class](https://github.com/Zodsmar/SeedSearcherStandaloneTool/blob/03486cd56f79d7939c23bb7dfd99fd5ed3ff868c/src/main/java/sassa/searcher/BiomeSearcher.java#L31), you can see that the search area is not circle, but a square, stretching from -radius to +radius in both the x and z directions.
Because both loops start at -radius, and the loop incrementing the z coordinate is inside the loop incrementing the x coordinate, you can think of the searcher going top-to-bottom then left-to-right, on a map where -Z (north) is up.
